### ISSUE TITLE
Enable EBE

### ISSUE THREAD

**Issue by IronOxidizer** [IronOxidizer/FastAsFabric#2]
EBE no longer requires a patched version of sodium.

https://github.com/FoundationGames/EnhancedBlockEntities/discussions/7#discussioncomment-832672

Remember to add EBE-supported blocks to EntityCulling config whitelist to avoid extra path tracing.

**Comment by IronOxidizer** [comment]
EBE relies on Indium which still isn't publishing release builds on modrinth, curseforge, or github, so we'll wait until builds are available before testing.

**Comment by Sirbucket** [comment]
I believe you can compile indium manually and ship that with the modpack.

**Comment by Sirbucket** [comment]
It's relatively easy.

**Comment by IronOxidizer** [comment]
> It's relatively easy.

Yes, but I want every Fast-as-Fabric build to be easily reproducible without requiring manual build for any of the dependent packages. Build artifacts don't count as easily reproducible either.

edit:
Indium now has builds available on GitHub and Modrinth

**Comment by IronOxidizer** [comment]
EBE will continue to be disabled for 1.17 as there's a bug with Indium causing FPS to be halved
